Orange/tests/test_statistics.py

```diff
@@ -158,3 +158,12 @@ def test_nanmean(self):
             np.testing.assert_array_equal(
                 nanmean(X_sparse),
                 np.nanmean(X))
+
+    def test_var(self):
+        for data in self.data:
+            for axis in range(len(data.shape)):
```

Comment:
Should we add `axis=None` to also test calculating variance on the whole matrix?